packages/oas-to-har/src/index.js

```diff
@@ -88,40 +88,39 @@ module.exports = (
     har.url = `https://try.readme.io/${har.url}`;
   }
 
-  if (!pathOperation.parameters) {
-    // eslint-disable-next-line no-param-reassign
-    pathOperation.parameters = [];
+  // Does this operation have any parameters?
+  const parameters = [];
```

Comment:
So this means we're creating a fresh array of parameters each time this is called? This is good for now, but eventually it would be nice to wrap this logic into here as operation.getParameters() or something, then it could be cached in there and not built out every time: https://github.com/readmeio/oas/blob/3e1983fb7b3bf04c1ba0f9286db6e0c158af8314/src/oas.js#L7

Reply:
Yeah that's what'll happen for now. I'd like to at some point refactor most of this HAR library since we're doing some similar stuff throughout it: https://github.com/readmeio/api-explorer/blob/fix/dupe-request-headers/packages/oas-to-har/src/index.js#L123

Could easily replace that with a `operation.getQueryParameters()` function call.